giskard/scanner/scanlogger.py

```diff
@@ -0,0 +1,56 @@
+import enum
+from typing import Optional
+
+from giskard.utils.xprint import (
+    Template,
+    xprint,
+)
+
+class ScanLoggerLevel(enum.Enum):
+    DEBUG = 0
+    INFO = 5
+    CRITICAL = 10
+
+class ScanLogger:
+    level: ScanLoggerLevel = ScanLoggerLevel.INFO
+    
+    def setLevel(self, level: str):
```

Comment:
let's create a type for level similar to what we did here for modeltypes:
```py
class SupportedModelTypes(Enum):
    CLASSIFICATION = "classification"
    REGRESSION = "regression"
    TEXT_GENERATION = "text_generation"


ModelType = Union[SupportedModelTypes, Literal["classification", "regression", "text_generation"]]
```
in your case:
```py
class ScanLoggerLevel(enum.Enum):
    DEBUG = 0
    INFO = 5
    CRITICAL = 10


LoggerLevel = Union[ScanLoggerLevel, Literal["DEBUG", "INFO", "CRITICAL"]]
```
and use `LoggerLevel` for the typing hints